src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -214,6 +214,13 @@ protected function describeContainerDefinition(Definition $definition, array $op
             }
         }
 
+        $calls = $definition->getMethodCalls();
+        if (count($calls) > 0) {
+            foreach ($calls as $callData) {
+                $output .= "\n".'- Call: `'.$callData[0].'`';
```

Comment:
OK, the current proposal is consistent with the display of tags, so this is fine